src/math/PhysicalConstants.h

```diff
@@ -1,7 +1,14 @@
 #pragma once
 
+#define _USE_MATH_DEFINES
+#include <math.h>
+
 namespace maxwell {
 namespace physicalConstants {
 	constexpr double speedOfLight = 1.0;
+	const double true_vacuum_permeability = 4.0 * M_PI * 1e-7;
```

Comment:
I would call this vacuumPermeabilitySI. It's more a matter of units rather than being true or false

Reply:
Agreed.

Reply:
Could you change it?